libr/core/cmd_open.c

```diff
@@ -1479,6 +1480,10 @@ static int cmd_open(void *data, const char *input) {
 						r_config_set (core->config, "cfg.debug", "false");
 						ut64 orig_baddr = sdb_num_get (core->sdb, "orig_baddr", 0);
 						r_bin_set_baddr (core->bin, orig_baddr);
+						ut64 baddr = r_config_get_i (core->config, "bin.baddr");
```

Comment:
bin.baddr should die, we should avoid using it

Reply:
Is there an alternative?

Reply:
not yet

Reply:
i mean you can get the current bin and check the baddr of it by using the r_bin_get_cur() api, its probably better than relay on bin.baddr, the thng is that having a variable is useful for the user as input to change the baddr to any other value. we should probably decide what to do in here before , tehre are other 16  places in core that this var is used

Reply:
this does not seem to be used at all, am i right?

Reply:
You are right! fixed